### ISSUE TITLE
get_string causes compile errors in newest deployed version

### ISSUE THREAD

**Issue by curiouskiwi** [cs50/libcs50#93]
Update50'd to v100 and now get_string causes compile errors.

```
   ~/workspace/ $ make word
clang -fsanitize=integer -fsanitize=undefined -ggdb3 -O0 -std=c11 -Wall -Werror -Wextra -Wno-sign-compare -Wshadow    word.c  -lcrypt -lcs50 -lm -o word
word.c:7:16: error: missing field 'prompt' initializer [-Werror,-Wmissing-field-initializers]
    string s = get_string();
               ^
/usr/include/cs50.h:139:68: note: expanded from macro 'get_string'
#define get_string(...) get_string(&(struct prompt) {0, __VA_ARGS__})
                                                                   ^
1 error generated.
make: *** [word] Error 1
```

**Comment by dmalan** [comment]
Ack, thanks, @curiouskiwi.

@crossroads1112, this relates to this is because we use `-Wextra and `-Werror`. We could resolve with `-Wno-missing-field-initializers` in `CFLAGS`, but that wouldn't solve this for psets using already-shipped `Makefile`s. Alternatively there's https://stackoverflow.com/a/9293087 but only for `clang`. Thoughts on what might work for `gcc` too?

**Comment by crossroads1112** [comment]
This is resolved by #95, no?
